history-deleter/history.js

```diff
@@ -14,7 +14,10 @@ function set_domain(domain) {
 }
 
 function no_history(hostname) {
-  document.getElementById('history').innerHTML = `No history for ${hostname}.`;
+  var history_text = document.getElementById('history');
+  while(history_text.firstChild )
+	history_text.removeChild(history_text.firstChild);
+  history_text.appendChild(document.createTextNode("No history for " + hostname + ".") );
```

Comment:
* It would be cool if you kept using template strings.
* Also, why not just use `.textContent`?
* Did you check `hostname` is just plain-text?